tests/integration/Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM golang:1.11 as builder
+FROM golang:1.9.7 as builder
+
 ARG CGO_ENABLED=0
 WORKDIR /go/src/github.com/newrelic/nri-cassandra
 COPY . .
 RUN make clean compile
+RUN wget https://download.newrelic.com/infrastructure_agent/binaries/linux/noarch/nrjmx_linux_1.5.2_noarch.tar.gz \
```

Comment:
Can you replace 1.5.2 by pipeline env var  NRJMX_VERSION: '1.5.2'? I have it in prerelease.yml